### PR TITLE
Main section mobile version

### PR DESCRIPTION
**Pull request by aliatayee**
second part (Main content) mobile version skeleton:
main-content structure for the second part has been defind defined.
styles applied in style.css.
images and static files added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,6 +47,113 @@ <h4>
                 <img src="assets/images/Disabled.svg" alt="">
             </div>
         </section>
+        <section id="my-recent-work">
+            <h2 class="title 2">My Recent<br>Work<br></h2>
+            <div class="indicator"></div>
+            <div class="img-patholder">
+                <div class="img-box"></div>
```
Comment (nicupop729) [inline index.html]: Inside the `div` tags with the class of `img-box`, we should have an image placeholder. So an `img` tag with a plain image that should keep the place of our project image.

- [ ] please make sure to implement the `img` tag inside your `div` with the class of `img-box`.

File: assets/css/style.css [hunk 2]
```diff
@@ -124,6 +136,202 @@ h4.slogan {
...
+.get-my-resume button {
+  background-color: #36b37f;
+  border: none;
+  padding: 20px;
+  font-size: 18px;
+  color: #fff;
+  margin-top: 15px;
+  margin-bottom: 50px;
+  border-radius: 5px;
+}
```
Comment (nicupop729) [inline assets/css/style.css:296]: See project buttons and Get my resume button should follow the follow the button styling: Active, Hover and Pressed (see design)
image

- [ ] Kindly I ask to implement the styling to all your buttons as shown in the design

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @aliatayee , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] The links in your README file are not working. Kindly ask to improve your README file with this aspect.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aliatayee** [comment]
thank you very much
sure !

**Comment by aliatayee** [comment]
@nicupop729 I have submitted the project please check it out

**Review by AnselemOdims — APPROVED** [review]
### Hi  @aliatayee 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aliatayee** [comment]
thank you very much!

**Pull request merged**